.github/workflows/zowe-cli-plugin.yml

```diff
@@ -74,7 +74,7 @@ jobs:
 
```

Comment:
we may also want to update `t1m0thyj/unlock-keyring@master` to `t1m0thyj/unlock-keyring@v1`